x/concentrated-liquidity/position_test.go

```diff
@@ -2071,3 +2071,92 @@ func (s *KeeperTestSuite) TestGetAllPositionIdsForPoolId() {
 	s.Require().Equal(expectedPositionOneIds, positionOne)
 	s.Require().Equal(expectedPositionTwoIds, positionTwo)
 }
+
+func (s *KeeperTestSuite) TestCreateFullRangePositionLocked() {
+	var (
+		positionId         uint64
+		liquidity          sdk.Dec
+		concentratedLockId uint64
+		err                error
+	)
+
+	invalidCoinsAmount := sdk.NewCoins(DefaultCoin0)
+	invalidCoin0Denom := sdk.NewCoins(sdk.NewCoin("invalidDenom", sdk.NewInt(1000000000000000000)), DefaultCoin1)
+	invalidCoin1Denom := sdk.NewCoins(DefaultCoin0, sdk.NewCoin("invalidDenom", sdk.NewInt(1000000000000000000)))
+	zeroCoins := sdk.NewCoins(sdk.NewCoin(ETH, sdk.ZeroInt()), sdk.NewCoin(USDC, sdk.ZeroInt()))
+
+	tests := []struct {
+		name                  string
+		remainingLockDuration time.Duration
+		coinsForPosition      sdk.Coins
+		expectedErr           error
+	}{
+		{
+			name:                  "valid test",
+			remainingLockDuration: 24 * time.Hour * 14,
+			coinsForPosition:      DefaultCoins,
+		},
+		{
+			name:                  "invalid coin0 denom",
+			remainingLockDuration: 24 * time.Hour * 14,
+			coinsForPosition:      invalidCoin0Denom,
+			expectedErr:           types.Amount0IsNegativeError{Amount0: sdk.ZeroInt()},
+		},
+		{
+			name:                  "invalid coin1 denom",
+			remainingLockDuration: 24 * time.Hour * 14,
+			coinsForPosition:      invalidCoin1Denom,
+			expectedErr:           types.Amount1IsNegativeError{Amount1: sdk.ZeroInt()},
+		},
+		{
+			name:                  "invalid coins amount",
+			remainingLockDuration: 24 * time.Hour * 14,
+			coinsForPosition:      invalidCoinsAmount,
+			expectedErr:           types.NumCoinsError{NumCoins: len(invalidCoinsAmount)},
+		},
+		{
+			name:                  "edge: both coins amounts' are zero",
+			remainingLockDuration: 24 * time.Hour * 14,
+			coinsForPosition:      zeroCoins,
+			expectedErr:           types.NumCoinsError{NumCoins: 0},
+		},
+	}
+
+	for _, test := range tests {
+		test := test
+		s.Run(test.name, func() {
+			// Init suite for each test.
+			s.SetupTest()
+			DefaultJoinTime := s.Ctx.BlockTime()
+			s.Ctx = s.Ctx.WithBlockTime(DefaultJoinTime)
```

Comment:
What's the point of this?

Reply:
added some logic to test: https://github.com/osmosis-labs/osmosis/pull/5369/commits/084783ac887184496c4c6360eb5993f71c00d8ab.

basically, I use it to check position's `joinTime` field